bin/bi-local

```diff
@@ -122,18 +122,18 @@ do_dev() {
 
 do_int_test() {
     local tag
-    tag=$(bix d get-image-tag)
+    tag=$(bi-docker get-image-tag)
```

Comment:
If we use `bix`, we'll do an extra arg parse and case traversal. Skip it and call the desired binary directly.